components/inquiry_questionnaire/test/views/test_inquiry.coffee

```diff
@@ -83,7 +83,7 @@ describe 'Inquiry', setup ->
       @view.__serialize__.then =>
         # Sets up the inquiry
         @inquiry.get('message').should.equal 'I wish to buy the foo bar'
-        @inquiry.get('contact_gallery').should.be.true()
+        @inquiry.get('contact_gallery').should.equal 'true'
```

Comment:
Note that now that we set `contact_gallery` in the form, we are sending String 'true' instead of Boolean true.